ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -200,109 +205,145 @@ Item {
-                    }
+                RowLayout {
+                    id: communityDescriptionLayout
 
-                    StatusBaseText {
-                        id: communityDesc
-                        text: invitedCommunity.description
-                        Layout.leftMargin: root.innerMargin
-                        Layout.rightMargin: root.innerMargin
-                        Layout.fillWidth: true
-                        wrapMode: Text.WrapAtWordBoundaryOrAnywhere
-                        font.pixelSize: 15
-                        color: Theme.palette.directColor1
-                    }
+                    Layout.leftMargin: d.margin
+                    Layout.rightMargin: d.margin
+                    Layout.topMargin: 12
+                    Layout.bottomMargin: 12
 
-                    StatusBaseText {
-                        id: communityNbMembers
-                        // TODO add the plural support
-                        text: qsTr("%1 members").arg(invitedCommunity.nbMembers)
-                        Layout.leftMargin: root.innerMargin
-                        font.pixelSize: 13
-                        font.weight: Font.Medium
-                        color: Theme.palette.baseColor1
+                    spacing: 12
+
+                    StatusSmartIdenticon {
+                        Layout.alignment: Qt.AlignTop
+                        Layout.preferredWidth: 40
+                        Layout.preferredHeight: 40
+
+                        name: d.invitedCommunity.name
+
+                        asset {
+                            width: 40
+                            height: 40
+                            name: d.invitedCommunity.image
+                            color: d.invitedCommunity.color
+                            isImage: true
+                        }
                     }
 
-                    Separator {
-                        Layout.fillWidth: true
+                    ColumnLayout {
+                        spacing: 2
+
+                        StatusBaseText {
+                            Layout.fillWidth: true
+
+                            text: d.invitedCommunity.name
+                            font.weight: Font.Bold
+                            wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+                            font.pixelSize: 17
+                            color: Theme.palette.directColor1
+                        }
+
+                        StatusBaseText {
+                            Layout.fillWidth: true
+
+                            text: d.invitedCommunity.description
+                            wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+                            font.pixelSize: 15
+                            color: Theme.palette.directColor1
+                        }
+
+                        StatusBaseText {
+                            Layout.fillWidth: true
+
+                            // TODO add the plural support
```

Comment:
Just do it :)

```qml
qsTr("%n member(s)", "", d.invitedCommunity.nbMembers)
```

Reply:
Ah yes, will do, thanks :) (This comment was already there)